README.md

```diff
@@ -31,7 +31,7 @@ Now, to start your app run ```npm start``` in the rootpath of the project. Then
 ## Development
 
 #### Environments
-By default, the environment will be **development**, but you can easily change it using the **NODE_ENV** environmental variable.
+By default, the environment will be **development**, but you can easily change it using the **NODE_ENV** environmental variable. IIn this files you can configure a DB connection and the schema it should use (the default ispublic).
```

Comment:
Typo `IIn`

Reply:
Also `ispublic`

Reply:
I'll write something like `In this files you can set your DB connection and the schema that should use (default: public)`. Also, the files should be linked here.